lib/facebook_ads/ad_objects/server_side/custom_data.rb

```diff
@@ -211,7 +222,8 @@ def ==(o)
             predicted_ltv == o.predicted_ltv &&
             num_items == o.num_items &&
             status == o.status &&
-            search_string == o.search_string
+            search_string == o.search_string &&
+            custom_properties == o.custom_properties
```

Comment:
can we do equality on standard hash?

Reply:
@sheel yes we can https://ruby-doc.org/core-2.5.5/Hash.html#method-i-3D-3D
>Equality—Two hashes are equal if they each contain the same number of keys and if each key-value pair is equal to (according to Object#==) the corresponding elements in the other hash.

Reply:
awesome